presto-native-execution/presto_cpp/main/PrestoMain.cpp

```diff
@@ -37,3 +38,11 @@ folly::Singleton<facebook::velox::BaseStatsReporter> reporter([]() {
   return new facebook::velox::DummyStatsReporter();
 });
 #endif
+
+#ifndef PRESTO_MEMORY_CHECKER_TYPE
```

Comment:
Christian and I were wondering if this block can be moved inside `PeriodicMemoryChecker.cpp`

Reply:
Moved block inside PeriodicMemoryChecker.cpp, confirmed it works